src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java

```diff
@@ -30,7 +29,6 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
           content.setPreferredFocusableComponent(null);
           toolWindow.getContentManager().addContent(content);
           DefaultActionGroup customCodySettings = new DefaultActionGroup();
-          customCodySettings.add(new OpenPluginSettingsAction("Cody Settings..."));
```

Comment:
<img width="1006" alt="image" src="https://github.com/user-attachments/assets/cfdc0654-89dc-4a42-bacb-e8bfe7b4997f">

Probably this is not an item for this PR but our settings seems to be orphaned now 😢 Can we still have them under `Tools > Sourcegraph & Cody`?

Reply:
Thanks for spotting it!
It is definitely for this PR, in fact I removed too much (update channel selection).
I added it back now.